loops-and-arrays.js

```diff
@@ -3,35 +3,41 @@
 // Schrijf een script dat iedere naam in onderstaande array vervangt door een koosnaampje, door er -"je" achter te plakken.
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 namen zou bevatten!
 // ==========================================
-
-const names = ["Henk", "Piet", "Fred", "Joop"];
-
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+const names = ["Henk", "Piet", "Fred", "Joop"];
 
-
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i] = names[i] + "je");
+}
 
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-const numbers = [2, 4, 5, 29, 38];
-
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 4, 8, 15, 87, 76 ];
 
+const numbers = [2, 4, 5, 29, 38];
 
-
-
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] === 0) {
```

Comment:
ik snap nit waarom heb je dit gedaan. deze conditie is nit nodig. wat je is elseif en else en else heb gedaan was genoeg.